bench.py

```diff
@@ -37,48 +47,46 @@ def calculator(test, arg, n_runs, only_result, workpath=os.getcwd()):
             statistics.median(w),
             statistics.pstdev(w)),
         bold=True)
-    os.remove(os.path.join(workpath, "Benchmark.txt"))
 
+    if uploadresult is True:
+        codespeedinfo.uploadresult(test, w)
 
-@click.group()
-def cli():
-    """A benchmark suite for Scrapy."""
-    pass
+    os.remove(os.path.join(workpath, "Benchmark.txt"))
 
 
-@cli.command()
+@click.group()
 @click.option(
     '--n-runs',
     default=1,
     help="Take multiple readings for the benchmark.")
 @click.option('--only_result', is_flag=True, help="Display the results only.")
-def bookworm(n_runs, only_result):
+@click.option('--uploadresult', is_flag=True, help="Upload the results")
+def cli(n_runs, only_result, uploadresult):
+    """A benchmark suite for Scrapy."""
+    global run, result, upload
```

Comment:
looks like there is a way to pass this options without using global variables: http://click.pocoo.org/5/complex/#the-root-command